.envrc

```diff
@@ -16,7 +16,11 @@ env_dir=./.env
 [[ -d "$layout_dir" ]] || mkdir -p "$layout_dir"
 
 if [[ ! -d "$env_dir" || ! -f "$layout_dir/nix-rebuild" || "$store_paths" != $(< "$layout_dir/nix-rebuild" ) ]]; then
-  nix-build ./nix -A wireServer.devEnv --out-link ./.env
+  if [[ "${USE_NOM}" == "1" ]]; then
```

Comment:
nix-output-monitor is coming from outside the development environment, if the user did add it to their system config / user profile.

I'd probably invert the logic a bit - use `nom-build` if it's in `$PATH`, but let people override behaviour by setting `USE_NOM` to `0` if they want to opt-out.

Reply:
(and this can probably be moved into its own PR, so it can land independently of the GHC upgrade)

Reply:
I've added it to the devShell, it seems very non-nix-y to depend on the state of the environment to do something, idk; additionally idk if people want nom all the time, most times the entire environment is built and people want to just evaluate the nix and then be dropped in the devShell. Nom just displays a tiny stopwatch while evaluating so perhaps it will look like "it's not doing something". 
But if you think that it would make more sense like that, I'll change it. :)

Reply:
But you're not guaranteed to be in the devshell already (assuming a clean checkout).

Hence the proposal - pick nom from `$PATH`, if people have it in their environment and didn't explicitly opt out with `USE_NOM=0`, but not add it to the dev environment (as it's likely already there)

Reply:
Makes sense. Yes I will implement it like that then 😌